snap/info_snap_yaml.go

```diff
@@ -72,7 +72,8 @@ type appYaml struct {
 	Command      string   `yaml:"command"`
 	CommandChain []string `yaml:"command-chain,omitempty"`
 
-	Daemon string `yaml:"daemon"`
+	Daemon     string     `yaml:"daemon"`
+	DaemonMode DaemonMode `yaml:"daemon-mode"`
```

Comment:
I spent some more time thinking about this name,  mode really applies to systemd management daemon itself because is the same code running in two different places/ways. My preference is to call this daemon-scope and change other things accordingly.

Reply:
`daemon-scope` sounds great.  I've adjusted the branch to match.